.gitignore

```diff
@@ -5,3 +5,4 @@ phpunit.xml
 composer.phar
 package.tar
 /packages.json
+/.phpunit
```

Comment:
Ok, clear

side:
> make the logic locating phpunit much more complex

well.. that doesn't sounds valid, if it is in require dev it will always be in one location, so you can scrap the logic involved for that and rather focus on other things, but if it is not possible this is besides the point. nothing to see here :walking: